setup.cfg

```diff
@@ -1,13 +1,29 @@
 [flake8]
-exclude=**/migrations/*,wsgi.py,__init__.py,**/node_modules/*
```

Comment:
wsgi.py and __init__.py no longer being excluded,  is this correct?

Reply:
It is. I figured they didn't throw any style errors, so it didn't make sense to exclude them from the checks!